FEM/rf_num_new.cpp

```diff
@@ -116,7 +116,7 @@ CNumerics::CNumerics(string name)
 		cpl_error_tolerance[i] = -1.0; // JT2012: should not default this. Should always be entered by user!
 	//
 	// ELE
-	ele_gauss_points = 3;
+	ele_gauss_points = 2;
```

Comment:
In the previous versions,  ele_gauss_points is fixed to 2 for line element and 3 for quad and hex.
Just changed it to 3. Now all benchmarks with line element get longer computation time and better accuracy in solutions than before.